webofneeds/won-owner-webapp/src/main/webapp/app/components/atom-connections-indicator.jsx

```diff
@@ -14,15 +14,18 @@ import "~/style/_atom-connections-indicator.scss";
 const mapStateToProps = (state, ownProps) => {
   const atom = getIn(state, ["atoms", ownProps.atomUri]);
 
-  const requestedConnections = atomUtils.getRequestedConnections(atom);
+  const requestedConnections = atomUtils.getRequestedConnections(atom, state);
   const requestsCount = requestedConnections ? requestedConnections.size : 0;
   const unreadRequests =
     requestedConnections &&
     requestedConnections.filter(conn => conn.get("unread"));
   const unreadRequestsCount = unreadRequests ? unreadRequests.size : 0;
 
   const hasUnreadChatMessages = ownProps && ownProps.hasUnreadChatConnections;
-  const unreadChatConnections = atomUtils.getUnreadChatMessageConnections(atom);
+  const unreadChatConnections = atomUtils.getUnreadChatMessageConnections(
```

Comment:
i think both methods can be moved to either general or connection selectors, i get that they would fit in the atom utils, but we would probably run into a cyclic dependency that way...